src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/editors/media.html

```diff
@@ -7,9 +7,9 @@
 
 	<div ng-if="control.value">
 		<img
-			ng-if="url"
+			ng-if="getThumbnailUrl()"
```

Comment:
Same problem mentioned above, using methods in binding expressions is expensive, it means these methods fire on every digest whereas when it's a property value a watch is setup and it's only executed when the value changes, since angular cannot detect a value change on a method its fired every time which is why these were set to properties. For best practices in angular we should not be binding to methods unless we need to